test/test_packages/SandboxFallback2/Project.toml

```diff
@@ -0,0 +1,9 @@
+name = "Foo"
+uuid = "02250abe-2050-11e9-017e-b301a2b5bcc4"
+authors = ["David Varela <00.varela.david@gmail.com>"]
```

Comment:
Maybe remove this and the other `authors` fields?

Reply:
Sure, I always forget to do that.

Reply:
But he wants to leave his mark 😁

Reply:
Deezy wuz here